pkg/master/ports/ports.go

```diff
@@ -17,7 +17,13 @@ limitations under the License.
 package ports
 
 const (
-	// KubeletPort is the default port for the kubelet status server on each host machine.
+	// KubeletStatusPort is the default port for the kubelet healthz server.
+	// May be overridden by a flag at startup.
+	KubeletStatusPort = 10248
```

Comment:
Should we use this define in Kubelet? (the dependency is not one we expect, but there is value in standardizing the source for the default ports.)